crates/ruma-client-api/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ Improvements:
   `IncomingAuthData::to_outgoing` on it.
 * Add custom variant to `LoginInfo` which can be constructed with `IncomingLoginInfo::new` and
   then call `IncomingLoginInfo::to_outgoing` on it.
+* Fix deserialization of `r0::session::get_login_types::CustomLoginType` and add test for it.
```

Comment:
This should be in a separate `Bug fixes` section. Also, the test isn't really sth. users would care about I think.

```suggestion
* Fix deserialization of `r0::session::get_login_types::CustomLoginType`.
```